rnaseq/hisat2_align.cwl

```diff
@@ -12,7 +12,6 @@ requirements:
 arguments: [
     "-p", $(runtime.cores),
     "--dta",
-    "--rna-strandness", "RF",
```

Comment:
Do we want to remove this? I'm not sure it has much effect, but all of our standard production RNA-seq is TruSeq Stranded Total RNA.

Reply:
I'm running a second test pipeline now so I can compare stranded and unstranded params on the same data (in kallisto and hisat). Will let you know if it matters much or not.

Reply:
reposting from slack:
I just did a comparison of stranded vs unstranded params on a small test data set.  The alignments are pretty much identical with HISAT2, but about 0.3% of genes have substantially altered expression via kallisto

So yeah, small difference in general, but I'm still more comfortable having the correct flags set (with the understanding that a mixup is not the end of the world)

Reply:
I agree, we could make it an optional parameter and then the higher level workflows would need to pass the appropriate value.